pkg/test/executor/test_runner.go

```diff
@@ -70,9 +71,19 @@ func RunTestCase(
 
 	resultsDirectory := t.TempDir()
 	runnerOutput, err := executor.Run(ctx, "test-execution", job, resultsDirectory)
-	require.NoError(t, err)
-	require.Empty(t, runnerOutput.ErrorMsg)
+	if err != nil {
+		if strings.Contains(err.Error(), `"media": executable file not found in $PATH`) {
+			t.Log("Expected error occured for no entrypoint", err)
+			//skip rest of test if correct error occurs
+			return
+		} else {
+			require.NoError(t, err)
+		}
+	} else {
+		require.Empty(t, runnerOutput.ErrorMsg)
+	}
```

Comment:
Is this still necessary? It would be preferable to not encode test-specific logic in this generic code.